docs/auto-setup.adoc

```diff
@@ -0,0 +1,111 @@
+= Automatic setup on virtual machine
+
+== Prerequisites
+
+You need to install:
+
+- https://www.packer.io/downloads[Packer]
+- https://www.vagrantup.com/downloads[Vagrant]
+- https://www.virtualbox.org/wiki/Downloads[VirtualBox]
+
+[TIP]
+VirtualBox installation may fail on macOS Catalina. If it's your case, please look
+at https://apple.stackexchange.com/questions/372492/virtualbox-installation-failed-on-macos-catalina[this issue].
+
+== Installing the system
+
+First, go to the `deployments/local-setup-environment` directory and build the environment image:
+```
+packer build -only=vagrant packerfile.json
+```
+An output directory named `vagrant-box` containing `package.box` file should be created.
+
+Then, switch the directory to `deployments/local-setup-instance` and import
+the image:
+```
+vagrant box add --force --name=local-setup-environment ./../local-setup-environment/vagrant-box/package.box
+```
+Finally, you can run the virtual machine by doing:
+```
+vagrant up
+```
+A new virtual machine will be created and provisioned accordingly. When
+this command terminates successfully, the machine should be ready to work
+and have all the auxiliary software like Geth, Bitcoin Core, ElectrumX
+and Core/ECDSA clients running.
+
+== How to interact with the system
+
+You can interact with the system through the tBTC dApp or automated
+end-to-end tests. Before you start interacting, you have to connect
+to the virtual machine. First, switch to the `deployments/local-setup-instance`
+directory and do:
+```
+vagrant ssh
+```
+
+You will be moved to the virtual machine home directory where you can find
+the `local-setup` directory where all installed things live.
+
+=== Keep Dashboard dApp
+
+To run the Keep Dashboard dApp invoke:
+```
+./run-keep-dashboard.sh
```

Comment:
Something is wrong:

```
vagrant@ubuntu-xenial:~/local-setup$ ./run-keep-dashboard.sh

Starting Keep Dashboard...

> dashboard@1.2.4-rc start /home/vagrant/local-setup/keep-core/solidity/dashboard
> npm run watch-css & npm run start-js


> dashboard@1.2.4-rc start-js /home/vagrant/local-setup/keep-core/solidity/dashboard
> react-scripts start


> dashboard@1.2.4-rc watch-css /home/vagrant/local-setup/keep-core/solidity/dashboard
> npm run build-css && less-watch-compiler src/css src/css app.less

sh: 1: react-scripts: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! dashboard@1.2.4-rc start-js: `react-scripts start`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the dashboard@1.2.4-rc start-js script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-07-22T11_58_57_963Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dashboard@1.2.4-rc start: `npm run watch-css & npm run start-js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dashboard@1.2.4-rc start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-07-22T11_58_57_982Z-debug.log
vagrant@ubuntu-xenial:~/local-setup$
> dashboard@1.2.4-rc build-css /home/vagrant/local-setup/keep-core/solidity/dashboard
> lessc --clean-css src/css/app.less src/css/app.css

sh: 1: lessc: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! dashboard@1.2.4-rc build-css: `lessc --clean-css src/css/app.less src/css/app.css`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the dashboard@1.2.4-rc build-css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-07-22T11_58_58_287Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dashboard@1.2.4-rc watch-css: `npm run build-css && less-watch-compiler src/css src/css app.less`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dashboard@1.2.4-rc watch-css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-07-22T11_58_58_307Z-debug.log
```

Reply:
Looks like something is wrong with the dashboard itself. I'll take a look.

Reply:
A long investigation is needed here. Most likely, it isn't related with this PR so I opened a separate issue for it: https://github.com/keep-network/local-setup/issues/15